Update/some_004.txt

```diff
@@ -3179,10 +3179,10 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#c8647c>暁</color>", NULL, "<color=#c8647c>Akira</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 39, "ps3/s04/39/310200040", 256, TRUE);
 	OutputLine(NULL, "「…それで、できれば公由にも、",
-			NULL, "\"...And so, if you can,", Line_WaitForInput);
+			NULL, "\"...And so, if it's possible,", Line_WaitForInput);
 	ModPlayVoiceLS(4, 39, "ps3/s04/39/310200041", 256, TRUE);
 	OutputLine(NULL, "その…好きになってもらえたらって…」",
-			NULL, " could... you say if you like me too...?\"", Line_Normal);
+			NULL, " Uh... I want you to say you love me too...\"", Line_Normal);
```

Comment:
```suggestion
			NULL, " uh... I want you to say you love me too...\"", Line_Normal);
```
Previous line ends with a comma rather than a period or ellipsis, so next line needs to start with a lowercase letter.